### PR TITLE
netbox in A

### PR DESCRIPTION
**Pull request by tthoma24**
Add Netbox A record

### CODE REVIEW

File: mesh.zone [hunk 1]
```diff
@@ -18,6 +18,7 @@ jim	A 10.70.140.70
 mail	A 10.70.140.70
 donuts A 10.70.73.29
 unifi A 10.70.95.63
+netbox A 10.70.95.135
```
Comment (leee) [inline mesh.zone:21]: `10.70.90.135` not `10.70.95.135` - `10.70.95.0/24` is sn3 oob.

**Review by leee — CHANGES_REQUESTED** [review]

**Review by leee — APPROVED** [review]
LG now, thanks. Someone else will have to merge though.

File: mesh.zone (hunk 1 above)
Comment (tthoma24) [reply mesh.zone]: Fixed in 514c37c

**Comment by zgiles** [comment]
Can/Will merge. Just as a note, merging doenst mean I agree that Netbox should be the canonical data source, but I am OK with and dont want to block your use of DNS.. no issue there..

**Pull request merged**